src/application.ts

```diff
@@ -155,7 +155,7 @@ export default class Application {
     const resourceClass = await this.resourceFor(data.type);
     const schemaRelationships = resourceClass.schema.relationships;
 
-    Object.keys(data.relationships).forEach(relationshipName => {
+    Object.keys(data.relationships).filter(relationshipName => data.relationships[relationshipName]).forEach(relationshipName => {
```

Comment:
@spersico These filters could simply be `.filter(Boolean)`

Reply:
okay, I'll give it a go

Reply:
@joelalejandro in https://github.com/ebryn/jsonapi-ts/pull/90/files#diff-8f42631f8d7d8cbaf47074d3ea5536d1R173 that worked, but here it breaks. Surely because one gets the keys (Object.keys), and the other the values (Object.values).
Even then, remember that this PR overlaps the #88, so when that one gets merged, part of this code will change.
I'll re-review it again then.

Reply:
It's not that important, we can iterate on this later, @spersico.